packages/terra-menu/tests/wdio/menu-spec.js

```diff
@@ -81,7 +81,7 @@ describe('Menu', () => {
       browser.click('#selectable-menu-button');
       browser.hasFocus('[class*="content"][aria-modal="true"][role="dialog"]');
       browser.click('.TestGroupItem3');
-      browser.hasFocus('li:last-child[aria-selected="true"][role="menuitemradio"]');
+      browser.hasFocus('li:last-child[aria-checked="true"][role="menuitemradio"]');
```

Comment:
I'm unfamiliar, what caused the change from aria-selected to aria-checked? List's implementation?

Reply:
It was a failing test that we didn't catch until list broke navigation.